apps/example-todo-app/tests/openapi-generation/openapi-generation.test.ts

```diff
@@ -79,4 +79,6 @@ test("generateOpenAPI correctly parses description with front matter", async (t)
     "This endpoint allows you to add a new todo item to the list. Deprecated."
   )
   t.is(routeSpec.deprecated, "Use foobar instead.")
+  t.is(routeSpec["x-fern-sdk-return-value"], "foobar")
+  t.is(routeSpec.response_key, "foobar")
```

Comment:
Is there an expected standard for openapi metadata? Based on `x-fern-sdk-return-value` it appears metadata should follow HTTP header format. If that`s true, then I'd expect `response-key` not `response_key`.  Can we confirm?

[The `x-` prefix is done](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers), so no need for that

> Custom proprietary headers have historically been used with an X- prefix, but this convention was deprecated in June 2012 because of the inconveniences it caused when nonstandard fields became standard in

Reply:
Is [this what we are using](https://swagger.io/docs/specification/openapi-extensions/)? Accodrin to this we should name fields like `x-response-key`.

Reply:
Oh nice call! Yeah, seems like we have to prefix custom extensions with `x-`. I'll update the code.

Reply:
cc872cc45fd3dcd968d2f3d3b73f28c447e5f240, 3dc32dbd4ae7195ff8a0a1923bf9c53e01cd1a84